alf/networks/value_networks.py

```diff
@@ -175,7 +175,7 @@ def __init__(self,
 
         super().__init__(
             input_tensor_spec=value_network.input_tensor_spec, name=name)
-        self._encoding_net = value_network._encoding_net.make_parallel(n)
+        self._encoding_net = value_network._encoding_net.make_parallel(n, True)
```

Comment:
I thought this was intended instead of a bug. There are many other places where make_parallel doesn't allow non-parallel input.

Reply:
Similar to parallel CriticNetwork, parallel ValueNetwork is often called with non-parallel input.